docs/3.api/1.components/4.nuxt-link.md

```diff
@@ -25,6 +25,23 @@ In this example, we use `<NuxtLink>` component to link to another page of the ap
 </template>
 ```
 
+### Handling 404s
+
+When using `<NuxtLink>` for `/public` directory files or when pointing to a different app on the same domain, you should use the `external` prop.
+
+Using `external` forces the link to be rendered as an `a` tag instead of a Vue Router `RouterLink`.
+
+```vue [pages/index.vue]
+<template>
+  <NuxtLink to="/the-important-report.pdf" external>
```

Comment:
It's possible that we implement this logic automatically for files, not sure if Vue Router handles file links at all?

Reply:
We can definitely switch it to use `a` if the `download` attribute is set which I plan to do in a separate PR

Reply:
FYI vue-router doesn't differentiate any kind of link. They are all meant to be resolved with `router.resolve()`

Reply:
I see. Not sure the initial thinking of the original `NuxtLink` design but I'd like to not see this error being thrown for valid links.

> [Vue Router warn]: No match found for location with path "/the-important-report.pdf"

Guessing their may be some minor performance improvements by not using RouterLink for links that don't need them (?).